satellite-software/include/Logging.h

```diff
@@ -58,6 +59,18 @@
 #define PRINT_HEX_ERROR(x)      
 #define PRINTLN_ERROR(x)        
 
+#elif LOG_LEVEL == LOG_LEVEL_DEBUG_ONLY
+
+#define PRINT_DEBUG(x)          Serial.print(x)
+#define PRINT_HEX_DEBUG(x)      Serial.print(x, HEX)
+#define PRINTLN_DEBUG(x)        Serial.println(x)
+#define PRINT_INFO(x)           
+#define PRINT_HEX_INFO(x)       
+#define PRINTLN_INFO(x)         
+#define PRINT_ERROR(x)          
+#define PRINT_HEX_ERROR(x)      
+#define PRINTLN_ERROR(x)        
+
```

Comment:
What is the scenario where you would want DEBVUG messages but not the error messages?

Reply:
The entire program (when using the BME680) is way to large to fit in the flash memory when you want some form of logging. By only having DEBUG messages, the size is small enough and still a lot of info during normal operations is printed. 
The errors are only useful when something breaks while DEBUG messages are useful when testing.